README.md

```diff
@@ -27,6 +27,36 @@ Installers
 Refer to the [downloads page][etcher] for the latest pre-made
 installers for all supported operating systems.
 
+#### Debian Repository
+
+- Save the following as `/etc/apt/sources.list.d/etcher.list`:
+
+```
+deb https://dl.bintray.com/resin-io/debian stable etcher
+```
+
+- Trust Bintray.com's GPG key:
```

Comment:
Good point! I've tried making this change, however, even after adding numbering, the markdown seems to get rendered incorrectly. For example:

```
diff --git a/README.md b/README.md
index b4746a9..5191237 100644
--- a/README.md
+++ b/README.md
@@ -29,19 +29,19 @@ installers for all supported operating systems.

 #### Debian Repository

-- Save the following as `/etc/apt/sources.list.d/etcher.list`:
+1. Save the following as `/etc/apt/sources.list.d/etcher.list`:

 ```
 deb https://dl.bintray.com/resin-io/debian stable etcher
 ```

-- Trust Bintray.com's GPG key:
+2. Trust Bintray.com's GPG key:

 ```
 sudo apt-key adv --keyserver pgp.mit.edu --recv-keys 379CE192D401AB61
 ```

-- Update and install:
+3. Update and install:

 ```
 sudo apt-get update
```

Yields:

<img width="657" alt="screenshot 2016-11-07 08 28 27" src="https://cloud.githubusercontent.com/assets/2192773/20048294/2f84f8a2-a4c4-11e6-9bf9-241fa9f515e8.png">

Any ideas?